lib/jsdom/living/xmlhttprequest.js

```diff
@@ -590,6 +594,30 @@ module.exports = function createXMLHttpRequest(window) {
     DONE: 4
   });
 
+  function parseContentType(contentType) {
```

Comment:
Actually, this would all be a lot easier if maybe we just used https://www.npmjs.com/package/contenttype :))